actionlib_tutorials/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ generate_messages(DEPENDENCIES std_msgs actionlib_msgs)
 
 ## Define catkin exports
 catkin_package(
-  CATKIN_DEPENDS roscpp actionlib
```

Comment:
We could remove `roscpp` from package.xml (I assume it's transitively depended on by `actionlib`)?